scripts/typescript_checks.py

```diff
@@ -82,12 +835,91 @@ def compile_and_check_typescript(config_path: str) -> None:
     assert process.stdout is not None
     error_messages = list(iter(process.stdout.readline, ''))
 
-    if error_messages:
-        print('Errors found during compilation\n')
-        print('\n'.join(error_messages))
-        sys.exit(1)
+    if config_path == STRICT_TSCONFIG_FILEPATH:
+        # Generate file names from the error messages.
+        errors = [x.strip() for x in error_messages]
+        # Remove the empty lines and error explanation lines.
+        prefixes = ('core', 'extension', 'typings')
```

Comment:
This seems like a fragile way to filter out the error message lines of interest because the folder names could change down the road. Is there a way to parse the lines without depending on folder names?

Reply:
One more way to find line of interest is by using python regex for format **ts(120,5): => ts(x,y):** but i prefer use of ('core', 'extension', 'typings'), as after completion of strict typescript we are going to remove all those changes of this PR and use only one tsconfig file, inbetween we have no plan of change main parent folders name.

Reply:
I feel like the regex approach is more robust without much of a downside, but I'm fine with the current approach if @vojtechjelinek agrees

Reply:
I'm fine with both